src/components/Trade/hooks/useSwapper/useSwapper.ts

```diff
@@ -116,20 +120,28 @@ export const useSwapper = () => {
     })
 
     if (!minimumQuote) return
+    //use gas estimate from quote if estimatedGasFees var is not populated yet
+    const quoteEstimatedGasFees = fromBaseUnit(minimumQuote?.feeData?.fee || 0, feeAsset.precision)
```

Comment:
`0` is a valid `fee`.

```suggestion
    const quoteEstimatedGasFees = fromBaseUnit(minimumQuote?.feeData?.fee ?? 0, feeAsset.precision)
```